composer/core/types.py

```diff
@@ -33,15 +33,11 @@
     from typing import Protocol
 
 __all__ = [
-    "Batch", "BatchPair", "BatchDict", "PyTorchScheduler", "JSON", "MemoryFormat", "as_batch_dict", "as_batch_pair",
+    "Batch", "PyTorchScheduler", "JSON", "MemoryFormat",
     "DataLoader", "BreakEpochException"
 ]
 
-# For BatchPar, if it is a list, then it should always be of length 2.
-# Pytorch's default collate_fn returns a list even when the dataset returns a tuple.
-BatchPair = Sequence[Union[torch.Tensor, Sequence[torch.Tensor]]]
-BatchDict = Dict[str, torch.Tensor]
-Batch = Union[BatchPair, BatchDict, torch.Tensor]
+Batch = torch.Tensor
```

Comment:
```suggestion
Batch = Any
```

Reply:
done in [acf116c](https://github.com/mosaicml/composer/pull/954/commits/acf116c968f0458ddffd28ab9a4ad25b98c7b182)